packages/ia-zendesk-help-widget/src/icon-question-mark.js

```diff
@@ -0,0 +1,14 @@
+import { html } from 'lit-element';
+
+export default html`
+  <svg
+    height="20"
+    viewBox="0 0 20 20"
+    width="20"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <g fill="#fff" fill-rule="nonzero">
```

Comment:
The `fill` attribute should be replaced with a class of `fill-color` to adhere to our icon standard of setting the fill and stroke colors from outside of the markup. The CSS variables `--iconFillColor` and `--iconStrokeColor` should be set and used to provide the values for fill and stroke properties in the CSS for the component.